bootstrap/apollo-env.sh

```diff
@@ -14,3 +16,10 @@ if [[ -z "${color_start-}" ]]; then
   declare -r color_norm="${color_start}0m"
 fi
 
```

Comment:
What do you think about providing an option for ensuring they are disable too like:
```
if [[ "${APOLLO_LOG}" == "0" ]]; then
unset TF_LOG
unset ANSIBLE_LOG
fi
```
Should we also provide a level system so:
APOLLO_LOG=1 -> -v
APOLLO_LOG=2 -> -vv
APOLLO_LOG=3 -> -vvv
APOLLO_LOG=4 -> -vvvv

Then if we unset whatever there is in TF_LOG and ANSIBLE_LOG Applies